step-api/src/main/java/org/jenkinsci/plugins/workflow/steps/StepDescriptor.java

```diff
@@ -110,7 +111,7 @@ public Step newInstance(Map<String,Object> arguments) throws Exception {
      * @throws UnsupportedOperationException if this descriptor lacks the ability to do such a calculation
      */
     public Map<String,Object> defineArguments(Step step) throws UnsupportedOperationException {
-        return DescribableHelper.uninstantiate(step);
+        return DescribableModel.uninstantiate_(step);
```

Comment:
:ant: an oddly-named method. I would just delete it. Here it is not cumbersome to write

```java
return new DescribableModel(step.getClass()).uninstantiate(step);
```

and for the usage in `InputStepTest` it is easy enough to hardcode `InputStep.class`.